docs/sources/project/advanced-contributing.md

```diff
@@ -58,69 +71,69 @@ The following provides greater detail on the process:
     
 3. Talk to the community about your idea.
 
-	We have lots of <a href="../get-help" target="_blank">community forums</a>
-	where you can get feedback on your idea. Float your idea in a forum or two
-	to get some commentary going on it.
-	
+    We have lots of <a href="../get-help/" target="_blank">community forums</a>
+    where you can get feedback on your idea. Float your idea in a forum or two
+    to get some commentary going on it.
+
 4. Fork `docker/docker` and clone the repo to your local host.
 
 5. Create a new Markdown file in the area you wish to change.  
 
-	For example, if you want to redesign our daemon create a new file under the
-	`daemon/` folder. 
+    For example, if you want to redesign our daemon create a new file under the
+    `daemon/` folder. 
+
+6. Name the file descriptively, for example `redesign-daemon-proposal.md`.
 
-6. Name the file descriptively, for example `redesign-daemon-proposal.md`.	
-	
 7. Write a proposal for your change into the file.
 
-	This is a markdown file that describes your idea. Your proposal
-	should include information like:
-	
-	* Why is this changed needed or what are the use cases?
-	* What are the requirements this change should meet?
-	* What are some ways to design/implement this feature?
-	* Which design/implementation do you think is best and why?
-	* What are the risks or limitations of your proposal?
-	
-	 This is your chance to convince people your idea is sound. 
-	  
+    This is a Markdown file that describes your idea. Your proposal
+    should include information like:
+
+    * Why is this changed needed or what are the use cases?
```

Comment:
There is a line introduced in the list.  Also, the terminating sentence is part of the numbered list not the bulleted list.

![image](https://cloud.githubusercontent.com/assets/1347209/6541684/e21419ca-c493-11e4-8957-f0959dd16e60.png)